### ISSUE TITLE
Create issue template each developer/code repo 

### ISSUE THREAD

**Issue by dockerymick** [agstack/agstack-website#11]
## Create an informative issue (See each section)

Using the current documentation as a model, please complete the issue template. 

### Prerequisite: Start with what we have

Always look at the current website to describe the change that is needed. Use the following link: 

**[AgStack website](https://agstack.github.io/agstack-website/)**

### Describe the changes in the doc and link to your dev story

To promote open collaboration during code changes/updates

Provide info for the following steps:

1. - [ ] **Mandatory** Choose the page that needs to be updated:
     
      - [ ] About
      - [ ] Contribute
      - [ ] Projects (specify project name) 
      - [ ] APIs
      - [ ] Resources
      - [ ] Glossary
      - [ ] New topic

2. - [ ] **Mandatory for bugs:** Clearly define what the problem is, what the change is, and link to the page:

3. - [ ] **Mandatory for content:**
            
       - [ ] Add steps and/or other important conceptual information here: 
       - [ ] Add verification at the end of the task, how does the user verify success (a command to run or a result to see?)
       - [ ] Add link to related dev story, if applicable

**Create and collaborate on developer issues; create PR and receive approval/community agreement**
must create a `.github` folder to host issue templates
<img width="1568" alt="Screen Shot 2022-12-15 at 2 10 19 PM" src="https://user-images.githubusercontent.com/60616885/207946745-ae4a24a4-ded0-4dda-a44d-1ce6f8903080.png">

**Comment by dockerymick** [comment]
Considering my bandwidth and team progress, this is lower priority. Not moving this to Jira and closing the issue
